lua/macro-to-lua.md

```diff
@@ -562,45 +592,53 @@ end
 There is also repeat until:
 ```lua
 repeat
-  print('Loop #'..loop)
+  printf('Loop #%d', loop)
   loop = loop - 1
 until loop == 0
 ```
 
 ### /goto
 This is only mentioned here because it is commonly used in macros. Avoid using this in Lua.
 
-While Lua also has goto, it should be possible to handle most situations with normal flow control like those described above.
+While Lua also has goto, it should be possible to handle most situations with normal flow control like those described above.  
+
+One valid scenario for goto would be to handle the lack of `continue` in Lua, however, this can still be avoided in most situations.
 
 ## MQ
-The first line to most lua scripts written for MQ will be `local mq = require 'mq'` which provides access to all the MQ functionality.
+The first two lines to most lua scripts written for MQ will be:  
+```lua
+---@type Mq

```

Comment:
I'm pretty sure that this `---@type` line is no longer needed with the most recent changes